pmix_governance.tex

```diff
@@ -449,6 +449,33 @@ \subsubsection{Release Schedule Guidance}\label{release-schedule-guidance}}
 Release Managers will prioritize major releases of the standard over waiting for
 additional changes that are in progress but not yet accepted by the ASC.
 
+\hypertarget{administrative-changes-guidance}{%
+\subsubsection{Minor Document Changes Guidance}\label{administrative-changes-guidance}}
+The intention of the rigorous process outlined in 
+Section~\ref{the-standardization-process} for proposed
+changes to the PMIx Standard is to ensure quality and consistency of 
+the interface and the overall document. However, in some cases, 
+PMIx participants may propose minor changes to the document that 
+should not require the full process required for substantive changes. 
+Examples of these minor changes include fixes for typographical errors 
+(``typos''), missing or erroneous punctuation, and formatting changes.
+
+In the case minor changes are proposed, the proposer should 
+create an issue and pull request as described in Sections~\ref{initial-discussion}
+and \ref{proposed-modification} and then bring the
+changes to the attention of the ASC Co-Chairs and Release Managers.
+The Co-Chairs and Release Managers will evaluate the proposed
+changes to determine whether they are minor or substantive.
+If all Co-Chairs and Release Managers determine that the
+changes are minor and do not alter the meaning of the current standard text, 
+then the changes do not require the full process and
+the changes can be applied to the document in the next release.
+Alternatively, if any of the Co-Chairs or Release Managers determine 

```

Comment:
Do we want to allow community members (or ASC members) to also chime in with concern over how substantial a change is?  Or keep it limited to Co-Chairs and RMs?